src/SprykerEco/Yves/Econda/Theme/default/components/molecules/econda-tracker/econda-tracker.twig

```diff
@@ -1,43 +1,64 @@
+{% extends model('component') %}
 
-    <input type="hidden" name="econda_tracking_content" value="{{ content }}">
-    {% if query is defined %}
-        <input type="hidden" name="econda_search_query_string" value="{{ query }}">
+{% define config = {
+    name: 'econda-tracker',
+    tag: 'econda-tracker'
+} %}
+
+{% define data = {
```

Comment:
So nothing is required in this molecule?

Reply:
Yes, the statistic send by this molecule depends on data passed into it, therefore, the set of installed data properties may differ